contracts/TokenVault.sol

```diff
@@ -178,6 +189,35 @@ contract TokenVault is Ownable, Recoverable {
     return token.balanceOf(address(this));
   }
 
+  /// @dev Check how many tokens "investor" can claim
+  /// @param investor Address of the investor
+  /// @return uint How many tokens the investor can claim now
+  function getClaimableAmount(address investor) public constant returns (uint claimableAmount) {
+    uint maxTokensLeft = balances[investor] - claimed[investor];
+    uint sinceLastClaim = lastClaimedAt[investor];
+    uint maxClaim;
+
+    if (tokensPerSecond > 0) {
+      if (sinceLastClaim == 0) {
+        sinceLastClaim = freezeEndsAt;
+      }
+
+      if (now < freezeEndsAt) {
```

Comment:
One thing that needs to be decided too is *whether* this should be `now < freezeEndsAt` or `now <= freezeEndsAt` for the *non-tappable case*. Both are used now / or the opposite of `now <= freezeEndsAt` in the state getter.

Reply:
I think "<" is fine, because that's the condition of not-claiming. If it passes claim(), getCurrentlyClaimableAmount() should return maxTokensLeft.